### ISSUE TITLE
Marching orders

### ISSUE THREAD

**Issue by tessam30** [tessam30/Ethiopia#18]
From our meeting today.
__Outstanding data visualization, colorization, datatization tasks for Ethiopia LAM:__
- [x] Create folders on the shared drive  
- [x] Investigate color palette options @flaneuse 
- [x] Create .mxds of rasters and cluster analysis in ArcMap for exporting to AI @pgault 
- [x] Create standardized regression coefficients 
- [x] Look into creating graphics using marginal effects at the margins
- [x] Determine aspect ratios and dimensions for graphics @pgault 
- [x] Create .mxd of Religion data (point data); Maybe create four of the same, rest in gray except highlighted religion? @pgault or tessam
- [x] Create graphic/one-pager for Mobile phones and wealth and WASH variables @flaneuse 
- [x] Bring Tim iced coffee daily @pgault

**Comment by tessam30** [comment]
Anything to add for Mr. Brent?

**Comment by flaneuse** [comment]
uhh pretty sure you mean **Dr.** Brent.  

What about all the mapping tasks?

**Comment by bmccusker** [comment]
As my kids say "ooooh burn"

**Comment by bmccusker** [comment]
Yes, Dr. Tim. I need to create the interpolated surface for Any Shocks. Here is the list of map products I compiled: 
Ethiopia Stories
1. Price Shocks
2. hazard shocks
3. Health shocks
4. Food Consumption Score
5. Diet Diversity
6. Any Shock (do interpolation). 
7. FtF Basmaps with elevation and Land Scan
8. Regions Basemap
9. Religion Map
10. Cell phone between males and female

**Comment by tessam30** [comment]
> Any Shock (do interpolation).

Use the rptShShock variable for this; Only use households in both panels.

**Comment by tessam30** [comment]
> Create folders on the shared drive

Folders in the AI folder on the shared google drive. Please transfer relevant __finalized__ data visualizations, maps, puppets, etc.. to this drive.

**Comment by tessam30** [comment]
> Create standardized regression coefficients

These will be tough to create and interpret because of the large number of categorical or binary variables we have in the model. Generally, standardized beta coefficients are used to show the effect of a 1 unit change in standard deviation for a given covariate. This concept doesn't really make much sense w/ categorical or binary variables. Will focus on comparing and contrasting geographic fixed-effects and/or coefs for which the standardized coefs make sense.
